docs/get_started/plan_deployment/traditional_deployment/cluster_consideration/clus_maint.md

```diff
@@ -1,30 +1,32 @@
 # Cluster maintenance
 
-Maintaining HCL Portal in a cluster typically means applying corrective services \(fix packs and interim fixes\) or updating the software release level on each node in the cluster.
+Maintaining HCL Portal in a cluster typically means applying fix packs and interim fixes or updating the software release level on each node in the cluster.
```

Comment:
Maintaining HCL DX in a cluster typically means applying fix packs and interim fixes or updating the software release level on each node in the cluster.

Reply:
Maybe let's leave the changing of HCL Portal to HCL DX to [DXQ-33845](https://jira.cwp.pnp-hcl.com/browse/DXQ-33845)?